CanonicalLtd/snappy-docs
# Update to generator@v2.1.0

Use the latest version of [generator](https://github.com/canonical-webteam/generator-canonical-webteam), with support for proxies, so it can be used to build resources on the jenkins server.

By reviewing this you're mostly reviewing https://github.com/ubuntudesign/phone-docs/pull/10 too - you might as well go on and review that PR :)

Fixes #114 

QA
--

Check run commands still work as expected:

``` bash
./run
./run test
./run clean
./run serve
./run watch
```
